features/org.wso2.carbon.identity.organization.management.core.feature/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <groupId>org.wso2.carbon.identity.organization.management.core</groupId>
```

Comment:
didn't we have a carbon feature for org.wso2.carbon.identity.organization.management.core?
If not, how it was packed to the IS by default as a feature earlier?

Reply:
Yes we have. But in order to bundle these resource files, another feature module was added. It was the convention in both OAuth repo and framework repo.

<img width="506" alt="Screenshot 2023-08-03 at 11 43 03" src="https://github.com/wso2-extensions/identity-organization-management-core/assets/35717390/15c119e0-a68b-4ab0-8d89-781deb5cf810">

Also, with this effort, previous feature module was refactored from
`org.wso2.carbon.identity.organization.management.core.server.feature` to `org.wso2.carbon.identity.organization.management.core.feature` in order to comply with the convensions used in the other repos.

Reply:
I also checked the code bases. we can just go with `org.wso2.carbon.identity.organization.management.core.server.feature` including the service + files
In other components, we have a separate feature to the backend component, a carbon UI-related component and a composite feature.
`<component name>.server.feature` is the feature name used to pack backend service + config files
Refer: https://wso2.com/library/tutorials/2014/03/how-to-create-a-feature-from-wso2-carbon-component/

Reply:
Fixed by - https://github.com/wso2-extensions/identity-organization-management-core/pull/74/commits/d83a5778490e205d969d0ffeec461db8f0e288f4